Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper.js

```diff
@@ -193,6 +193,28 @@ function create(createOptions) {
         .async('string')
         .then((str) => Promise.resolve(str));
     },
+
+    fetchBinary(instance = {}, url, options = {}) {
+      const path = removeLeadingSlash(url);
+      if (!ready) {
+        vtkErrorMacro('ERROR!!! zip not ready...');
+      }
+
+      if (options.compression) {
+        if (options.compression === 'gz') {
+          return zipRoot.file(path).then((data) => {
+            const array = pako.inflate(data).buffer;
+            return Promise.resolve(array);
+          });
+        }
+        return Promise.reject(new Error('Invalid compression'));
+      }
+
+      return zipRoot
+        .file(path)
+        .async('uint8array')
+        .then((data) => Promise.resolve(data));
```

Comment:
you have an option for arraybuffer, that might be better than uint8array.

Reply:
Okay! I think I went with `uint8array` because the gzip data is converted to that ([here](https://github.com/Kitware/vtk-js/blob/6e6515f947e9ee8d6c4e1d383310293f93d07d4a/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper.js#L157) and [here](https://github.com/Kitware/vtk-js/blob/6e6515f947e9ee8d6c4e1d383310293f93d07d4a/Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper.js#L182)). But I just gave `arraybuffer` a try and it seemed to work fine also. I'll switch over to that one.

Reply:
Fixed. It seems to work fine.